shared/menubar/files.desktop.js

```diff
@@ -85,16 +82,14 @@ type ShowAllProps = FileUpdatesHocProps & {|
 
 const FileUpdatesShowAll = (props: ShowAllProps) => (
   <Kb.Box2 direction="horizontal" fullWidth={true} centerChildren={false} style={styles.buttonContainer}>
-    <Kb.Button
-      label={
-        props.isShowingAll
-          ? 'Collapse'
-          : `+ ${(props.numUpdates - defaultNumFileOptionsShown).toString()} more`
-      }
-      onClick={props.onShowAll}
-      small={true}
-      type="Secondary"
-    />
+    {!props.isShowingAll && (
+      <Kb.Button
+        label={`+ ${(props.numUpdates - defaultNumFileOptionsShown).toString()} more`}
+        onClick={props.onShowAll}
+        small={true}
+        type="Secondary"
+      />
+    )}
```

Comment:
Did you mean to leave the containing box in case of `!props.isShowingAll`? It results in a `tiny` margin.

Reply:
👍🏻 Updated